packages/core/test/NumberField_test.js

```diff
@@ -209,7 +211,9 @@ describe("NumberField", () => {
           formData: 2,
         });
 
-        expect(node.querySelector(".field input").value).eql("2");
+        expect(node.querySelector(".field input").getAttribute("value")).eql(
```

Comment:
It's a bit strange, because if you try doing `.value` for this line of code, everything works. Any idea why it works for this test case, but not for the other test cases in this file?

Reply:
honestly no clue, if you want me to dig deeper before merge I can try to figure it out

Reply:
figured it out - apparently "2." is not a valid value in input number fields, but for some reason we still support it which causes some issues. that jsdom supported this before was faulty behavior.

![image](https://user-images.githubusercontent.com/2225828/115143072-6d304680-a045-11eb-89d9-a733b8ddedbf.png)

Reply:
i would honestly prefer to remove support for "2." since it is not compatible with the HTML standard, but I just added a comment for now.

Reply:
Great, thanks for looking into this!